README.md

```diff
@@ -13,7 +13,7 @@ You can learn more about the story behind Cartography in our [presentation at BS
 
 ## Installation
 
-Time to set up the server that will run Cartography.  Cartography _should_ work on both Linux and Windows servers, but bear in mind we've only tested it in Linux so far.
+Time to set up the server that will run Cartography.  Cartography _should_ work on both Linux and Windows servers, but bear in mind we've only tested it in Linux so far.  Cartography requires Python 3.
```

Comment:
Specify which rev of Python 3? The Neo4J driver requires at least 3.4, might want to check the other requirements.